src/main/common/utils/numericUtils.ts

```diff
@@ -10,7 +26,15 @@ export function toNumberOrUndefined (value: any): number {
   }
 
   const strVal: string = value && value.toString().trim()
-  const numberVal: number = strVal.length > 0 ? _.toNumber(strVal) : undefined
 
-  return isNaN(numberVal) ? undefined : numberVal
+  // convert to number
+  const numberVal = convert(strVal)
```

Comment:
You wouldn't need this comment if `convert` had a more accurate name and/or all types specified.